docker-compose.yml

```diff
@@ -3,85 +3,51 @@ version: "3"
 services:
   zk:
     image: zookeeper:3.4
-    ports:
-      - 2188:2188
     environment:
       ZOO_MY_ID: 1
       ZOO_PORT: 2188
 
   master:
-    image: mesosphere/mesos-master:1.2.1
-    network_mode: host
```

Comment:
Part of the benefit of `network_mode: host` is that I can browse to the mesos ui at http://<docker vm>:5050
Do you have ideas on how to do that here? Same will be true for slaves, IF you want to be able to browse stdout/stderr of containers in the mesos ui. (less important, but can be useful). We may be able to just run the master in host network, and slaves in bridge, and live with not accessing slaves via ui.

Reply:
So the mac-add-route + the ./get-ips.sh should give you a set of IPs to go to so you can see all the mesos UIs.